routes/api/contacts.js

```diff
@@ -50,4 +51,26 @@ router.put('/:contactId', validate.contactUpdateValidator, async (req, res, next
   }
 });
 
+router.patch('/:contactId/favorite', async (req, res, next) => {
+  const { contactId } = req.params;
+  const { favorite } = req.body;
+
+  if (!favorite) {
+      return res.status(400).json({ message: "missing field favorite" });
+  }
+
+  try {
+    const updatedContact = await Contact.updateContact(contactId, { favorite });
+
+    if (updatedContact) {
+      return res.status(200).json(updatedContact);
+    } else {
+      return res.status(404).json({ message: "Not found" });
+    }
```

Comment:
Taki trick
```suggestion
    if (updatedContact) return res.status(200).json(updatedContact);
      return res.status(404).json({ message: "Not found" });
```
Prawda że bardziej czytelniejsze?

Reply:
Return sprawi że kod nie pójdzie dalej.